include/affinity.win.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <assert.h>
+#include <pthread.h>
+#include <unistd.h>
+
+#include "affinity.h"
+
+void affinity_int_base_mask() {
+	static bool initialized = false;
+	if(initialized) return;
+
+	assert(GetProcessAffinityMask(GetCurrentProcess(), &irt_g_affinity_base_mask, &sys_affinity_mask) == 0 && "Error retrieving base affinity mask.");
+	initialized = true;
+}
+
+uint32 affinity_cores_available() {
+	affinity_int_base_mask();
+
+	int bit_mask_length = sizeof(native_cpu_set) * 8;
+	native_cpu_set power = 1;
+	uint32 count = 0;
+	for(uint32 i = 0; i < bit_mask_length) {
+		if((power & g_affinity_base_mask) != 0) { ++count; }
+		power = power << 1;
+	}
+}
```

Comment:
This entire Windows implementation is clearly broken (function doesn't return anything, loop doesn't terminate, where is `sys_affinity_mask` coming from..?); I'd feel a lot more comfortable merging this having Windows CI in place...

Reply:
so should I remove this implementation until we have a win CI working ?